modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1957,33 +1941,6 @@ If there is a suggested replacement, update to use the replacement instead.
 .Using id() function
```

Comment:
The confusing part here is that we throw 01N02, but actually, there is a replacement `elementId()` and it should probably throw 01N01 instead.

Reply:
If 01N02 is the correct code, I need to move it to the section "Deprecated features without a future replacement".

Reply:
There is a Trello card about this https://trello.com/c/7yJJyG5q/995-deprecated-function-id-needs-a-better-error-message. elementId is not a direct replacement, it works in some situations and in some not. This whole deprecation is a very debated topic 😓  I think it might end up having 01N00 once that is implemented. For now 01N02 is what it has in the code.

Reply:
OK, I understand. I'll move it under the Deprecated features without a future replacement section.
By the way, in the Cypher Manual, it said that id() is replaced by elementId(). Here https://neo4j.com/docs/cypher-manual/current/functions/#header-query-functions-scalar. I guess we need to update that as well. This card is from 27 Aug 2023 😄.